### ISSUE TITLE
DJV-1.2.4/lib/djvGraphics/Cineon.cpp not compile

### ISSUE THREAD

**Issue by perestoronin** [darbyjohnston/DJV#5]
```
[  9%] Building CXX object lib/djvGraphics/CMakeFiles/djvGraphics.dir/CineonPlugin.cpp.o
cd /var/tmp/portage/media-gfx/djv-1.2.4/work/djv-1.2.4_build/lib/djvGraphics && /usr/bin/x86_64-pc-linux-gnu-g++ -D-DOPENEXR_DLL -DDJV_ASSERT -DDJV_LINUX -DDJV_MMAP -DFFMPEG_FOUND -DJPEG_FOUND -DOPENEXR_FOUND -DPNG_FOUND -DTIFF_FOUND -DdjvGraphics_EXPORTS -I/var/tmp/portage/media-gfx/djv-1.2.4/work/DJV-1.2.4/lib -I/var/tmp/portage/media-gfx/djv-1.2.4/work/DJV-1.2.4/bin -I/var/tmp/portage/media-gfx/djv-1.2.4/work/DJV-1.2.4/tests -I/var/tmp/portage/media-gfx/djv-1.2.4/work/djv-1.2.4_build -I/usr/include/glm -isystem /usr/include/qt5 -isystem /usr/include/qt5/QtCore -isystem /usr/include/qt5/QtDBus -isystem /usr/include/qt5/QtGui -isystem /usr/include/qt5/QtNetwork -isystem /usr/include/qt5/QtOpenGL -isystem /usr/include/qt5/QtSql -isystem /usr/include/qt5/QtSvg -isystem /usr/include/qt5/QtWidgets -isystem /usr/include/qt5/QtXml -isystem /usr/include/OpenEXR   -fPIC -Wno-write-strings -Wno-format-security -fPIC   -std=gnu++11 -o CMakeFiles/djvGraphics.dir/CineonPlugin.cpp.o -c /var/tmp/portage/media-gfx/djv-1.2.4/work/DJV-1.2.4/lib/djvGraphics/CineonPlugin.cpp
<command-line>:0:1: ошибка: имена макросов должны быть идентификаторами
[  9%] Building CXX object lib/djvGraphics/CMakeFiles/djvGraphics.dir/CineonSave.cpp.o
<command-line>:0:1: ошибка: имена макросов должны быть идентификаторами
cd /var/tmp/portage/media-gfx/djv-1.2.4/work/djv-1.2.4_build/lib/djvGraphics && /usr/bin/x86_64-pc-linux-gnu-g++ -D-DOPENEXR_DLL -DDJV_ASSERT -DDJV_LINUX -DDJV_MMAP -DFFMPEG_FOUND -DJPEG_FOUND -DOPENEXR_FOUND -DPNG_FOUND -DTIFF_FOUND -DdjvGraphics_EXPORTS -I/var/tmp/portage/media-gfx/djv-1.2.4/work/DJV-1.2.4/lib -I/var/tmp/portage/media-gfx/djv-1.2.4/work/DJV-1.2.4/bin -I/var/tmp/portage/media-gfx/djv-1.2.4/work/DJV-1.2.4/tests -I/var/tmp/portage/media-gfx/djv-1.2.4/work/djv-1.2.4_build -I/usr/include/glm -isystem /usr/include/qt5 -isystem /usr/include/qt5/QtCore -isystem /usr/include/qt5/QtDBus -isystem /usr/include/qt5/QtGui -isystem /usr/include/qt5/QtNetwork -isystem /usr/include/qt5/QtOpenGL -isystem /usr/include/qt5/QtSql -isystem /usr/include/qt5/QtSvg -isystem /usr/include/qt5/QtWidgets -isystem /usr/include/qt5/QtXml -isystem /usr/include/OpenEXR   -fPIC -Wno-write-strings -Wno-format-security -fPIC   -std=gnu++11 -o CMakeFiles/djvGraphics.dir/CineonSave.cpp.o -c /var/tmp/portage/media-gfx/djv-1.2.4/work/DJV-1.2.4/lib/djvGraphics/CineonSave.cpp
<command-line>:0:1: ошибка: имена макросов должны быть идентификаторами
<command-line>:0:1: ошибка: имена макросов должны быть идентификаторами
<command-line>:0:1: ошибка: имена макросов должны быть идентификаторами
make[2]: выход из каталога «/var/tmp/portage/media-gfx/djv-1.2.4/work/djv-1.2.4_build»
make -f tests/djvCoreTest/CMakeFiles/djvCoreTest.dir/build.make tests/djvCoreTest/CMakeFiles/djvCoreTest.dir/build
make[2]: вход в каталог «/var/tmp/portage/media-gfx/djv-1.2.4/work/djv-1.2.4_build»
[  9%] Building CXX object tests/djvCoreTest/CMakeFiles/djvCoreTest.dir/BoxTest.cpp.o
cd /var/tmp/portage/media-gfx/djv-1.2.4/work/djv-1.2.4_build/tests/djvCoreTest && /usr/bin/x86_64-pc-linux-gnu-g++ -DDJV_ASSERT -DDJV_LINUX -DDJV_MMAP -DPNG_FOUND -DdjvCoreTest_EXPORTS -I/var/tmp/portage/media-gfx/djv-1.2.4/work/DJV-1.2.4/lib -I/var/tmp/portage/media-gfx/djv-1.2.4/work/DJV-1.2.4/bin -I/var/tmp/portage/media-gfx/djv-1.2.4/work/DJV-1.2.4/tests -I/var/tmp/portage/media-gfx/djv-1.2.4/work/djv-1.2.4_build -I/usr/include/glm -isystem /usr/include/qt5 -isystem /usr/include/qt5/QtCore -isystem /usr/include/qt5/QtDBus -isystem /usr/include/qt5/QtGui -isystem /usr/include/qt5/QtNetwork -isystem /usr/include/qt5/QtOpenGL -isystem /usr/include/qt5/QtSql -isystem /usr/include/qt5/QtSvg -isystem /usr/include/qt5/QtWidgets -isystem /usr/include/qt5/QtXml   -fPIC -Wno-write-strings -Wno-format-security -fPIC   -std=gnu++11 -o CMakeFiles/djvCoreTest.dir/BoxTest.cpp.o -c /var/tmp/portage/media-gfx/djv-1.2.4/work/DJV-1.2.4/tests/djvCoreTest/BoxTest.cpp
make[2]: *** [lib/djvGraphics/CMakeFiles/djvGraphics.dir/build.make:264: lib/djvGraphics/CMakeFiles/djvGraphics.dir/Cineon.cpp.o] Ошибка 1
make[2]: *** Ожидание завершения заданий…

```

Please help to complete ebuild for version 1.2.4

**Comment by perestoronin** [comment]
src_configure() {
	local mycmakeargs=(
		-DCMAKE_INSTALL_PREFIX=/usr
		-DDJV_THIRD_PARTY=/usr/$(get_libdir)
		-DBUILD_SHARED_LIBS=ON
		-DCMAKE_VERBOSE_MAKEFILE=ON
	)

	cmake-utils_src_configure
}

**Comment by darbyjohnston** [comment]
Hi, thanks for the bug report; it looks like the error messages you included are incomplete? Are there any other error messages after this line?

<command-line>:0:1: errors...```

**Comment by perestoronin** [comment]
and djv 1.1.2 compile without any errors, but djv 1.2.4 with same ebuild does not.

**Comment by darbyjohnston** [comment]
Thanks for updating the error messages, I found the problem. There was a bug in the CMake files when enabling shared libraries; I hadn't noticed it before since I generally use static linking. I've checked the fix into the "release/1.2" branch.

**Comment by darbyjohnston** [comment]
Hi,

I just released a new version which has this fix, it is tagged as 1.2.5 in "release/1.2".
